content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -27,57 +44,47 @@ api = "0.5"
 
 # Buildpack ID and metadata
 [buildpack]
-id = "examples/ruby"
-version = "0.0.1"
-name = "Ruby Buildpack"
+  id = "examples/ruby"
+  version = "0.0.1"
 
 # Stacks that the buildpack will work with
 [[stacks]]
-id = "io.buildpacks.samples.stacks.bionic"
+  id = "io.buildpacks.samples.stacks.bionic"
+
 ```
 
 You will notice two specific fields in the file: `buildpack.id` and `stack.id`. The buildpack ID is the way you will reference the buildpack when you create buildpack groups, builders, etc. The stack ID is the root file system in which the buildpack will be run. This example can be run on one of two different stacks, both based upon Ubuntu Bionic.
 
 ### `detect` and `build`
 
-Next you will need to create the `detect` and `build` scripts. These files must exist in a `bin` directory in your buildpack directory.
+Next, we will cover the `detect` and `build` scripts. These files are created in `bin` directory in your buildpack directory.
 
-Create your `bin` directory and then change to that directory.
 
-<!-- test:exec -->
-```bash
-mkdir ruby-buildpack/bin
-```
-
-Now create your `ruby-buildpack/bin/detect` file and copy in the following contents:
+`ruby-buildpack/bin/detect` file has the following contents:
 
 <!-- test:file=ruby-buildpack/bin/detect -->
 ```bash
 #!/usr/bin/env bash
-set -eo pipefail
 
-exit 1
+exit 0
```

Comment:
```suggestion
exit 1
```

Reply:
i changed to status to exit 1, so it will give error for "$?"